docs/configuration.md

```diff
@@ -306,13 +306,16 @@ SQLPAD_LDAP_AUTH_ENABLED = "false"
 SQLPAD_LDAP_URL = ""
 
 # Base LDAP DN to search for users in
-SQLPAD_LDAP_BASE_DN = ""
+SQLPAD_LDAP_SEARCH_BASE = ""
 
 # Username for LDAP lookup
-SQLPAD_LDAP_USERNAME = ""
+SQLPAD_LDAP_BIND_DN = ""
 
 # Password for LDAP user used for LDAP lookup
 SQLPAD_LDAP_PASSWORD = ""
+
+# LDAP search filter. e.g. `(uid={{username}})` in OpenLDAP or `(sAMAccountName={{username}})` in ActiveDirectory
+SQLPAD_LDAP_SEARCH_FILTER = ""
```

Comment:
@dengc367 will this LDAP auth work if `SQLPAD_LDAP_SEARCH_FILTER` is empty?

Reply:
No, SQLPAD_LDAP_SEARCH_FILTER need setup which can not be empty, such as comment description for the LDAP/AD scenarios.

Reply:
Ah ok. I was curious because existing setups will need to update their configs for this to work properly. Luckily for us we have LDAP listed as experimental, so technically I think we're okay to make this change without making a major version change.